mandatory/4-tax.js

```diff
@@ -17,7 +20,10 @@ function calculateSalesTax() {}
   Remember that the prices must include the sales tax (hint: you already wrote a function for this!)
 */
 
-function addTaxAndFormatCurrency() {}
+function addTaxAndFormatCurrency(value) {
+  return "£" + (value*1.2).toFixed(2);
```

Comment:
Optional (or Consider): (value*1.2) = return value of function calculateSalesTax, better to declare a new variable to store the result of the function and use the variable to replace (value*1.2), so it would be more readible.

Reply:
Thank you very much Doris, I've updated my code.